src/makeInfinityAdder.js

```diff
@@ -5,6 +5,23 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const sumNumber = (number) => {
+    if (number === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+
+    sum += number;
+
+    return sumNumber;
```

Comment:
Returning `sumNumber` directly prevents the ability to chain calls because it will keep adding to the same sum. Instead, you should return a new function that can be called with the next number.